package.json

```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.2.0",
-    "@guardian/atom-renderer": "0.14.6",
+    "@guardian/atom-renderer": "^0.14.6",
```

Comment:
An insubstantial change given that the next version of `@guardian/atom-renderer` is `0.15.0`. Happy to go back to the specifying the exact version if necessary